src/test/java/net/snowflake/client/jdbc/DatabaseMetaDataIT.java

```diff
@@ -73,8 +73,8 @@ public void testDatabaseAndDriverInfo() throws SQLException {
       DatabaseMetaData metaData = connection.getMetaData();
 
       // JDBC x.x compatible
-      assertEquals(1, metaData.getJDBCMajorVersion());
-      assertEquals(0, metaData.getJDBCMinorVersion());
+      assertEquals(4, metaData.getJDBCMajorVersion());
```

Comment:
those assertions need to be moved to new test in `DatabaseMetaDataLatestIT` with comment marking above which version it works correctly
OldDrivers still reports 1.0 and fails on this assertion

Reply:
I've updated the tests as suggested but we now fail this check for DatabaseMetaDataIT.java with the updated code. Is that ok?

Reply:
No, it's not - we should remove those assertions you are changing from `DatabaseMetaDataIT`